examples/tinywl/main.cpp

```diff
@@ -61,6 +60,125 @@ WSurfaceItem *Helper::movingItem() const
     return m_movingItem;
 }
 
+bool Helper::registerExclusiveZone(WLayerSurface *layerSurface)
+{
+    auto exclusiveZone = layerSurface->exclusiveZone();
+    auto exclusiveEdge = layerSurface->getExclusiveZoneEdge();
```

Comment:
Where to update if `exclusiveZone` or `exclusiveEdge` is changed?

Reply:
In `refreshAnchors` will recheck exclusiveZone